### PR TITLE
fix(twitch): dev console, unthemed hr & vod chapters

### PR DESCRIPTION
**Pull request by mxgic1337**
## 🔧 What does this fix? 🔧

- Fixed unthemed hr in docs
fix
- Fixed wrong link color in dev console
fix2
- Fixed wrong text color on hover in VOD chapters
2024-11-12-085217_hyprshot

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Review by isabelroses — DISMISSED** [review]

**Comment by isabelroses** [comment]
Now that your a maintainer, if you get an approval you can self merge. 👍

File: styles/twitch/catppuccin.user.css
Comment (uncenter) [inline styles/twitch/catppuccin.user.css:635]: Hmm fading mantle? This should just be base/surface0/surface1 then since thats the shade you'll end up with I'd think.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/twitch/catppuccin.user.css
Comment (uncenter) [reply styles/twitch/catppuccin.user.css]: No I mean if you are fading a shade, like a non-accent color (crust, mantle, base, surface0, ..., subtext1, text) you should just use a different shade (unless you are fading it to a very low opacity for a different reason, such as an overlay on top of a video thumbnail).

Comment (uncenter) [reply styles/twitch/catppuccin.user.css]: So instead of `fade(@mantle, 80%)` or `fade(@base, 80%)`, use `@surface0`.

Comment (mxgic1337) [reply styles/twitch/catppuccin.user.css]: The background should be transparent, it is overlapping category cover art:
2024-11-12-164934_hyprshot

Comment (uncenter) [reply styles/twitch/catppuccin.user.css]: Oh gotcha so it isn't just behind the cover art. Sure, you can fade it. Go back to mantle if you want if you think that suits it better.

Comment (mxgic1337) [reply styles/twitch/catppuccin.user.css]: Actually, I think that the base color looks better.

**Review by uncenter — APPROVED** [review]

**Comment by uncenter** [comment]
As @isabelroses said now that you have received an approval by userstyles staff you can merge this PR yourself when you are ready.

**Pull request merged**
